### PR TITLE
Fix Docker container page look incompletely

### PR DESCRIPTION
**Pull request by Eji4h**
I fix my issue #62.

### CODE REVIEW

File: content/faq/docker/docker-container.md [hunk 1]
```diff
@@ -5,35 +5,51 @@ editable: true
 showToc: true
 ---
 
-# To list all container 
+# To list all container
```
Comment (dmakeroam) [inline content/faq/docker/docker-container.md]: list all containers? Plural?

File: content/faq/docker/docker-container.md [hunk 2]
```diff
@@ -5,35 +5,51 @@ editable: true
...
 ```
-# To run command in container 
+
+# To run command in container
+
 ```
 docker exec <container_id>
 ```
+
 # To copy files/folders between container and local filesytem
```
Comment (dmakeroam) [inline content/faq/docker/docker-container.md]: containers and local filesystem?

File: content/faq/docker/docker-container.md [hunk 3]
```diff
@@ -5,35 +5,51 @@ editable: true
...
 docker rm <container_id>
 ```
+
 # To fetch the logs
+
 ```
 docker logs <container_id>
 ```
+
 # To run command in container
```
Comment (dmakeroam) [inline content/faq/docker/docker-container.md]: To run a command?

**Review by dmakeroam — CHANGES_REQUESTED** [review]

File: content/faq/docker/docker-container.md (hunk 1 above)
Comment (Eji4h) [reply content/faq/docker/docker-container.md]: I don't change this. Just format.

Comment (Eji4h) [reply content/faq/docker/docker-container.md]: This is legacy wording. Sorry for only focus #62 issue.

**Comment by Eji4h** [comment]
I think maintainers can update the wording to be proper. I am not confident in my English and grammar skills. My PR is only focused on the #62 issue.

Thanks

**Comment by aaronamm** [comment]
@dmakeroam  I have fixed singular/plural nouns already. Thanks for your feedback.

**Comment by aaronamm** [comment]
@Eji4h Since this is a small PR (only one changed file), could you please squash all commit messages to one message.
Thanks.

image

**Pull request closed without merging**